server/deskroom/common/openai.py

```diff
@@ -20,8 +39,30 @@ def create_openai_client(asynchronous: bool = False) -> AsyncOpenAI | OpenAI:
     return create_openai_sync_client(api_key=settings.OPENAI_API_KEY)
 
 
+def create_azure_openai_client(
+    asynchronous: bool = False,
+) -> AsyncAzureOpenAI | AzureOpenAI:
+    if asynchronous:
+        return create_azure_openai_async_client(
+            api_key=settings.AZURE_OPENAI_API_KEY,
+            azure_endpoint=settings.AZURE_OPENAI_ENDPOINT,
+            api_version=settings.AZURE_OPENAI_API_VERSION,
+        )
+    return create_azure_openai_sync_client(
+        api_key=settings.AZURE_OPENAI_API_KEY,
+        azure_endpoint=settings.AZURE_OPENAI_ENDPOINT,
+        api_version=settings.AZURE_OPENAI_API_VERSION,
+    )
+
+
 def read_prompt(txt_file_from_prompts_dir: str | Path) -> str:
-    with open(PROMPT_PATH / txt_file_from_prompts_dir) as file:
-        prompt_file = file.readlines()
-    prompt = "\n".join(prompt_file)
+    if os.path.exists(txt_file_from_prompts_dir):
+        with open(PROMPT_PATH / txt_file_from_prompts_dir) as file:
+            prompt_file = file.readlines()
+        prompt = "\n".join(prompt_file)
+    else:
+        container_client = create_azure_container_client("prompts")
+        byte_data = container_client.download_blob(str(txt_file_from_prompts_dir))
+        prompt = byte_data.readall().decode("utf-8")
```

Comment:
로컬에서 파일을 한번 찾고 없으면 azure를 찾는 거 보다 명시적으로 환경 별로 구분하는 건 어떨까요?

```suggestion
    if setting.local():
        with open(PROMPT_PATH / txt_file_from_prompts_dir) as file:
            prompt_file = file.readlines()
        prompt = "\n".join(prompt_file)
    else:
        container_client = create_azure_container_client("prompts")
        byte_data = container_client.download_blob(str(txt_file_from_prompts_dir))
        prompt = byte_data.readall().decode("utf-8")
```